templates/zerver/app/index.html

```diff
@@ -167,6 +167,10 @@
     <source class="notification-sound-source-ogg" type="audio/ogg" />
     <source class="notification-sound-source-mp3" type="audio/mpeg" />
 </audio>
+<audio id="temp-realm-default-notification-sound-audio">
+    <source class="notification-sound-source-ogg" type="audio/ogg" />
+    <source class="notification-sound-source-mp3" type="audio/mpeg" />
+</audio>
```

Comment:
Hmm, this is oddly named -- how will the person notification settings page copy work?

Reply:
I have removed that extra ```<audio id="temp-realm-default-notification-sound-audio">``` tag and used the existing ```<audio id="realm-default-notification-sound-audio">```.